conf/nginx.conf

```diff
@@ -8,6 +8,11 @@ events {
 }
 
 http {
+  # Forward user's real ip in Cloudflare header.
+  # Avoid multiple proxy layers by using CF-Connecting-IP
+  # instead of X-Forwarded-For. 
+  real_ip_header CF-Connecting-IP;
```

Comment:
What happens if someone runs the container in an environment where this header (CF-Connecting-IP) is not set on incoming requests?

Reply:
It just defaults to the real ip which would be the aws ip.

Reply:
If by "aws ip" you mean the load balancer's IP, I think it might actually be the Docker network IP. When we were debugging the problem on Friday evening we were looking at the nginx access logs from the AWS EB-managed EC2 machine rather than nginx in the container. I guess that's another reason to do what you proposed and just configure nginx in the EB instance instead of running it in the container as well, so we don't have this confusion and also so that the IP addresses shown in the CloudWatch logs are correct.